test-cases/performance/perf-regression.100threads.30M-keys.yaml

```diff
@@ -13,12 +13,25 @@ n_db_nodes: 3
 n_loaders: 4
 n_monitor_nodes: 1
 
+#AWS
 instance_type_db: 'i3.2xlarge'
 instance_type_loader: 'c4.2xlarge'
 instance_type_monitor: 't3.small'
-
-user_prefix: 'perf-regression'
 space_node_threshold: 644245094
+#------
+#GCE
+gce_instance_type_db: 'n2-highmem-8'
+gce_root_disk_type_db: 'pd-ssd'
+gce_n_local_ssd_disk_db: 8
+gce_instance_type_loader: 'c3-highcpu-8'
+gce_root_disk_type_loader: 'pd-ssd'
+root_disk_size_loader: 50
+gce_instance_type_monitor: 'n2-highmem-8'
+#+ rewrite var from  perf-regression-throughput-config-gce-modifications.yaml
+#use_preinstalled_scylla: true
```

Comment:
Why not true?
I think that for GCE we must have it.

Reply:
in my test runs it does not work. will check again and fix it or back with error

Reply:
UPD for GCE default is "use_preinstalled_scylla: false" for AWS: "use_preinstalled_scylla: true"
so decided to set explicitly "use_preinstalled_scylla: true" in this config

Reply:
now both GCE and AWS will be set to True by default in #6363

Reply:
good will remove it